src/game.cpp

```diff
@@ -8842,7 +8842,7 @@ point game::look_around(WINDOW *w_info, const point pairCoordsFirst)
 }
 
 bool lcmatch(const std::string &str, const std::string &findstr); // ui.cpp
-bool game::list_items_match(item &item, std::string sPattern)
+bool game::list_items_match(item *item, std::string sPattern)
```

Comment:
Hmm, don't know why I changed that, went a little over board without thinking I guess.